pulsar-broker-common/src/main/java/org/apache/pulsar/broker/ServiceConfiguration.java

```diff
@@ -487,6 +487,13 @@ The delayed message index time step(in seconds) in per bucket snapshot segment,
     )
     private long brokerShutdownTimeoutMs = 60000;
 
+    @FieldContext(
+            category = CATEGORY_SERVER,
+            dynamic = true,
+            doc = "The max bundle unload concurrency limit when broker shutdown. Set to 0 means no limit."
+    )
+    private int brokerShutdownMaxBundleUnloadPerMinute = 0;
```

Comment:
Why is the unit in minutes

Reply:
Please see the discussion https://lists.apache.org/thread/7f5f9gn7qq86hstrkw5oz51g0rxrqq37